specs/src/specs/block_validity_rules.md

```diff
@@ -1 +1,98 @@
 # Block Validity Rules
+
+Unlike most blockchains, Celestia derives most of its functionality from
+stateless commitments to data rather than stateful transitions. This means that
+the protocol relies heavily on block validity rules. Notably, resource
+constrained light clients must be able to detect when these validity rules have
+not been followed in order to avoid making an honest majority assumption.
```

Comment:
[Question] Is the block validity rule the same as the logics implemented in the [ProcessProposal](https://github.com/celestiaorg/celestia-app/blob/8ae6a84b2c99e55625bbe99f70db1e5a985c9675/app/process_proposal.go#L20)? If we were to map it to a single function, would process proposal be the right one?

Reply:
Yup